.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
Node 8 has reached end of life. It's incompatible with the `@hapi/lab` used in this repo:
https://github.com/hapijs/lab/issues/980

Reply:
yeah, we only support 10+